packages/dialtone-vue2/components/emoji_picker/modules/emoji_search.vue

```diff
@@ -22,13 +22,13 @@
         <dt-button
           importance="clear"
           size="xs"
-          class="d-mrn4"
+          class="d-emoji-picker__search-x-button"
           circle
           kind="muted"
           @click="clearSearch"
         >
           <template #icon>
-            <dt-icon-close
+            <dt-icon-x-circle
```

Comment:
Is this icon change intended?

Reply:
I think it should be close instead of x-circle. My mistake.